plugins/woocommerce/changelog/52487-fix-add-to-cart-form-interactivity-api

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Add to Cart Form: Ensure Compliance with min, max, and step HTML Attributes.
```

Comment:
Personal preference, but I would use lowercase for everything that is not a proper noun:

```suggestion
Add to Cart Form: Ensure compliance with min, max, and step HTML attributes.
```